cmd/exo/cmd/ssh.go

```diff
@@ -107,18 +109,19 @@ func getSSHInfo(name string, isIpv6 bool) (*sshInfo, error) {
 
 }
 
-func printSSHConnectSTR(info *sshInfo) error {
+func printSSHConnectSTR(info *sshInfo) {
 
 	if _, err := os.Stat(info.sshKeys); os.IsNotExist(err) {
-		return fmt.Errorf("default ssh KeyPair not found %q infos.sshKeys", info.sshKeys)
+		log.Println("Warning: Identity file Default ssh keypair not found not accessible: No such file or directory.")
```

Comment:
what is not found?

Reply:
![image](https://user-images.githubusercontent.com/15922119/42807414-0558ab2c-89b1-11e8-9770-1ebcccee0b22.png)

Reply:
where wasn't the what not found? :monkey_face: